src/game.cpp

```diff
@@ -3022,8 +3022,7 @@ bool game::handle_action()
                     }
                     as_m.text = data.str();
                 }
-                if( (u.has_item_with_flag("ALARMCLOCK") || u.has_bionic("bio_watch")) &&
-                    !(u.hunger < -60) ) {
+                if  ( u.has_alarm_clock() && !(u.hunger < -60) ) {
```

Comment:
Unexpected problems getting an antique wooden cabinet moved through a questionably wide airlock mean I'm around tonight.  This is fixable.